### DISCUSSION TITLE
Dynamic skybox list (editing UI)

### DISCUSSION THREAD

**Discussion by Jigoku** [redeclipse/base#1178] [Ideas and Feature Requests]
I've just cobbled this together; a (potentially hackish) way of listing all the skyboxes in the editing UI, i'm sure it could be improved, but has the benefit of not needing to hardcode skybox paths/names. 

Here's the patch if anyone wanted to look at it further;
https://gist.github.com/Jigoku/6c15195216322ee2aa5a8cbf2e408bbc

(diff based on https://github.com/redeclipse/base/commit/6767f32711bd9eff1d9c42699c87dc992be08300)

![2019-10-07-212109_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/66345729-82082600-e948-11e9-8fbb-3fcbe9056ce9.png)

**Comment by lambda-11235** [comment]
What's wrong with the current skybox selection UI?

**Comment by no-lex** [comment]
It's entirely hardcoded.
